### ISSUE TITLE
training code

### ISSUE THREAD

**Issue by kaishijeng** [zylo117/Yet-Another-EfficientDet-Pytorch#1]
Do you have a plan to release it?
I want to train on a different dataset.

Thanks

**Comment by zylo117** [comment]
i am working on the map test, only when i confirm its performance, will i work on the training script.

it should be within a day or two.

**Comment by kaishijeng** [comment]
Great. Waiting for the release.

Thanks,

**Comment by zylo117** [comment]
> Great. Waiting for the release.
> 
> Thanks,

Hi, I open a new branch of trainig, can you try running it with you dataset?

I tried training it on coco with D0, it seems ok.

https://github.com/zylo117/Yet-Another-EfficientDet-Pytorch/blob/training/train.py

**Comment by lxyyang** [comment]
@zylo117 how to set the parameters，such as lr？

**Comment by zylo117** [comment]
> @zylo117 how to set the parameters，such as lr？

I update training code and merge into master, you can check out readme for more info.

# train efficientdet-d2 on a custom dataset with pretrained weights
# with batchsize 8 and learning rate 1e-5 for 10 epoches

python train.py -c 2 --batchsize 8 --lr 1e-5 --num_epochs 10

**Comment by kaishijeng** [comment]
I have downloaded the training branch and tested it. Unfortunately, it has encountered errors which I have created another issue ticket. Close this ticket.
